corehq/apps/auditcare/management/commands/generate_request_report.py

```diff
@@ -1,18 +1,41 @@
+import argparse
 import csv
+import gzip
+from textwrap import dedent
 
 from django.core.management.base import BaseCommand, CommandError
 
 from corehq.apps.domain.models import Domain
 from corehq.util.argparse_types import date_type
 
-from ...utils.export import get_users_to_export, write_log_events
+from ...utils.export import get_users_to_for_domain, write_log_events
 
 
 class Command(BaseCommand):
-    help = """Generate request report"""
+
+    help = dedent("""\
+        Generate request report of NavigationEventAudit events
+
+        NOTES
+            - AccessAudit events are not included in this report.
+            - All filtering options are AND'd together (important to consider when
+              combining both --user and --domain options).
+            - Specifying the --domain option excludes all non-domain navigation events
+              (admin activities, account/profile management, etc).
+            - Specifying the --domain option without the --user option will result in a
+              report limited to "users associated with that domain" and superusers.
```

Comment:
@snopoke, for my own understanding, what is an "enterprise user"?

Reply:
A user who has access to the domain not because they are a member of the domain but because they are a member of the 'enterprise domain' to which this domain is linked:

https://github.com/dimagi/commcare-hq/blob/f3dc008aacfb8e6b9f32411569162f9bd55dc5c8/corehq/apps/enterprise/models.py#L8-L13